R/vaccination.R

```diff
@@ -1,79 +1,59 @@
-build_rel_susceptibility <- function(rel_susceptibility) {
+build_rel_param <- function(rel_param, name_param = "rel_param") {
```

Comment:
my strong suggestion is to avoid default args on internal functions like this, as it leads to surprising bugs when you forget them